### ISSUE TITLE
[Bug]: RSS Rules are not read properly

### ISSUE THREAD

**Issue by Tobiyashi** [VueTorrent/VueTorrent#1550]
### Bug description

The "Edit RSS Rules" are not read and written properly from the download_rules.json file.
Before the release of VueTorrent 2.7 i could edit in the webui the download path for the RSS Rules.
But with update 2.7 and 2.7.1 the possibility to edit the folder disapeared. Instead the button "torrent parameter" apeared. Sadly the already changed and working download paths, which are present in the "download_rules.json" are not shown and the changes, which are made in the WebUi, aren't saved in the "download_rules.json".

Additionally the value for "last match" is not shown in the "Edit RSS Rule" window but shown in the "download_rules.json".

### Expectations

The values in the download_rules.json are shown in the "Edit RSS Rule" window and changes are saved

### Steps to reproduce

1. Add or change RSS Rule
2. Change Download Path in the "Torrent paramters" window
3. click close and save.
4. The rule is added but the changed folder is missing in the "download_rules.json"

### How did you install VueTorrent?

Fetched from latest-release branch

### Environment description

```plain text
Device type and name: 
OS: Ubuntu 22.04.4 LTS

PWA: Don't know what that means
Running inside a container: No
```

### qBittorrent version

4.5.5

### VueTorrent version

2.7.1

### Agreements

- [X] The bug still occurs on the nightly build
- [X] The bug still occurs on the latest version of qBittorrent
- [X] A similar issue doesn't already exist

**Comment by Larsluph** [comment]
Please include the associated requests sent, one for the "setRule" request and one for the "rules" request.

**Comment by Larsluph** [comment]
Tested on my setup using your reproduction steps and it works...

**Comment by Tobiyashi** [comment]
Thanks for the replies:

here is the content of the setRule:
```
ruleName: TestTest123
ruleDef: {"affectedFeeds":[],"enabled":true,"episodeFilter":"","ignoreDays":0,"lastMatch":"","mustContain":"TEST321","mustNotContain":"","name":"TestTest123","priority":0,"smartFilter":false,"useRegex":false,"previouslyMatchedEpisodes":[],"torrentParams":{"save_path":"/raid/torrent/TEST123","skip_checking":false,"add_to_top_of_queue":false,"content_layout":"Original","stop_condition":"None","download_limit":7168000,"upload_limit":2560000,"use_download_path":true,"download_path":"/raid/torrent/temp","stopped":false,"use_auto_tmm":false,"ratio_limit":-2,"seeding_time_limit":-2,"inactive_seeding_time_limit":-2}}
```

this is what got written in the sownload_rules.json:
```
     "TestTest123": {
        "addPaused": null,
        "affectedFeeds": [
        ],
        "assignedCategory": "",
        "enabled": true,
        "episodeFilter": "",
        "ignoreDays": 0,
        "lastMatch": "",
        "mustContain": "TEST321",
        "mustNotContain": "",
        "previouslyMatchedEpisodes": [
        ],
        "savePath": "",
        "smartFilter": false,
        "torrentContentLayout": null,
        "useRegex": false
    },
```
attached is the screenshot of the rule i created.
I feel like this is linked to another bug https://github.com/VueTorrent/VueTorrent/issues/1571

![Screenshot 2024-03-14 090923](https://github.com/VueTorrent/VueTorrent/assets/49440579/205848ab-291b-4b1e-bdeb-fe4f1cf32337)

**Comment by Tobiyashi** [comment]
if i open the rule again the path is back to the default one.
If i edit the download_rules to the correct value it works. But it doesn't show it in the webui.
![Screenshot 2024-03-14 091224](https://github.com/VueTorrent/VueTorrent/assets/49440579/1de8ded8-a117-4661-9574-e740f9afd4dc)

**Comment by Larsluph** [comment]
It looks like the torrentParams attribute isn't read.

I checked and it has been introduced in 4.6.0

**Comment by Tobiyashi** [comment]
Thanks for checking. Unfortunately qBitorrent 4.6.0+ is not available for Ubuntu 22.04 LTS through https://launchpad.net/~qbittorrent-team/+archive/ubuntu/qbittorrent-stable.
Is there a chance to bring back the rss-functionality before the torrentParams was introduced?

**Comment by Larsluph** [comment]
> Is there a chance to bring back the rss-functionality before the torrentParams was introduced?

What we can do is bind both the old and the new fields but keep in mind that some fields might not be used in some versions.
